### PR TITLE
Move to daily release

### PR DESCRIPTION
**Pull request by sdake**
move from master-latest-release to release-1.3-latest-daily

Please provide a description for what this PR is for.

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

[ ] Configuration Infrastructure
[ ] Docs
[ ] Installation
[ ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -27,9 +27,9 @@ ISTIO_DOCKER_HUB ?= docker.io/istio
 export ISTIO_DOCKER_HUB
 ISTIO_GCS ?= istio-release/releases/$(VERSION)
 ISTIO_URL ?= https://storage.googleapis.com/$(ISTIO_GCS)
-ISTIO_CNI_HUB ?= gcr.io/istio-release
+ISTIO_CNI_HUB ?= gcr.io/istio-testing
```
Comment (howardjohn) [inline Makefile]: this does not contain the daily builds, so this will break things

File: galley/pkg/source/kube/builtin/testdata/node.yaml [hunk 2]
```diff
@@ -131,7 +131,7 @@ status:
     sizeBytes: 446398900
   - names:
     - gcr.io/istio-release/proxyv2@sha256:dec972eab4f46c974feec1563ea484ad4995edf55ea91d42e148c5db04b3d4d2
-    - gcr.io/istio-release/proxyv2:master-latest-daily
+    - gcr.io/istio-release/proxyv2:release-1.3-latest-daily
```
Comment (howardjohn) [inline galley/pkg/source/kube/builtin/testdata/node.yaml]: nit - would rather we just leave these or make them something arbitrary like `tag` so we don't have to keep changing them. The tag here is meaningless, its just test data

File: Makefile (hunk 1 above)
Comment (howardjohn) [reply Makefile]: We publish daily builds to `gcr.io/istio-release`

File: galley/pkg/source/kube/builtin/testdata/node.yaml (hunk 2 above)
Comment (howardjohn) [reply galley/pkg/source/kube/builtin/testdata/node.yaml]: although I guess that would have to be a change on master

File: Makefile (hunk 1 above)
Comment (rlenglet) [reply Makefile]: @sdake

Comment (sdake) [reply Makefile]: @howardjohn daily builids are published to istio-release only for cni, or for all repos?  (my question is around the proxy push specifically).

**Pull request merged**